install_files/securedrop-ossec-agent/var/ossec/etc/ossec.conf

```diff
@@ -108,7 +108,7 @@
   <localfile>
     <log_format>full_command</log_format>
     <command>head -1 /var/lib/securedrop/submissions_today.txt | grep '^[0-9]*$'</command>
-    <frequency>86400</frequency>
+    <frequency>90000</frequency> <!-- 25 hours -->
```

Comment:
Question re: #3368 - does change mean that this `localfile` won't be considered a daily notification and thus `do_not_group` will not apply?

Reply:
Ah okay nevermind, I see that the corresponding rule in `local_rules.xml` is using `rule_id=400600` ✅